test/spmd/tensor/test_ops.py

```diff
@@ -102,6 +103,36 @@ def test_ones_like(self):
         ones_expected = torch.ones(4, 8)
         self.assertEqual(ones_expected, ones_like_dt.local_tensor())
 
+    def _run_sharded_elementwise_ops(
+        self, mesh, spec, input_size, op, **kwargs
+    ):
+        torch.manual_seed(self.rank)
+        input_tensor = torch.randn(*input_size, requires_grad=True)
+        dist_tensor = Tensor.from_local(input_tensor, mesh, spec)
+        dt = op(dist_tensor)
+        expected = op(input_tensor)
+        self.assertEqual(input_tensor, dist_tensor.local_tensor())
+        if "training" in kwargs and kwargs["training"]:
+            self.assertNotEqual(expected, dt.local_tensor())
+        else:
+            self.assertEqual(expected, dt.local_tensor())
+
+    @with_comms
+    def test_activations(self):
+        device_mesh = DeviceMesh(self.device_type, list(range(self.world_size)))
+        self._run_sharded_elementwise_ops(device_mesh, [Shard(0)], (8, 5), torch.nn.functional.gelu)
+        self._run_sharded_elementwise_ops(device_mesh, [Replicate()], (8, 5), torch.nn.functional.gelu)
+        self._run_sharded_elementwise_ops(device_mesh, [Shard(1)], (3, 14), torch.nn.functional.relu)
+        self._run_sharded_elementwise_ops(device_mesh, [Replicate()], (8, 5), torch.nn.functional.relu)
+
+    @with_comms
+    def test_dropout(self):
```

Comment:
can we only test this for GPU? also add a TODO here to enable CPU case so we don't forget haha

Reply:
Sure.